backend/src/main/java/com/gestionnaire_de_stage/exception/UnusableTokenException.java

```diff
@@ -0,0 +1,12 @@
+package com.gestionnaire_de_stage.exception;
+
+import lombok.Getter;
+
+@Getter
+public class UnusableTokenException extends Exception {
+    private final String message;
+
+    public UnusableTokenException(String message) {
+        this.message = message;
+    }
```

Comment:
```suggestion
public class UnusableTokenException extends Exception {

    public UnusableTokenException(String message) {
     super(message);
    }
```
Voir review précédant